src/styles/partials/components/_step-list.scss

```diff
@@ -71,14 +72,15 @@
     }
 
     &.is-hovered:before {
-      background: $blue-light;
+      background: $vibrant-blue;
       border-color: $vibrant-blue;
       color: $white;
     }
   }
 }
 
 .dg_step-list__toggle-btn__title {
+  margin-bottom: $margin-large;
```

Comment:
confirmed with @sdurphy this can be $margin-small

Reply:
done